packages/protobuf-test/typescript/tsconfig.4_1_2.json

```
@@ -1,6 +1,16 @@
 {
-  "include": ["../src/gen/js/extra/*.js", "../src/gen/ts/extra/*.ts"],
-  // These are the default compiler options for TypeScript v4.1.2, created with `tsc --init`
+  "include": ["../src/**/*"],
+  "exclude": [
+    // Exclude WKTs and their usage in tests because they use syntax not
+    // compatible with older versions of TypeScript.  WKTs used by actual
+    // consumers of this library are generated with ts-nocheck so its safe
+    // to ignore them here and still get version compatibility assurance.
```

I think our ts-nocheck would only help the other way round - if we were missing an `override`. Perhaps reword like so?:

```suggestion
    // Exclude WKTs and their usage in tests because they use syntax not
    // compatible with older versions of TypeScript.  WKTs are typically not
    // generated, but imported from @bufbuild/protobuf, which is compatible.
    // Users of this version of TypeScript must use them.
```